fastfield_codecs/src/lib.rs

```diff
@@ -63,6 +63,7 @@ pub trait FastFieldDataAccess {
 }
 
 #[derive(Debug, Clone)]
+/// Contains statistics used for compression in field writing.
```

Comment:
I assumed so. Might be a little incorrect.

Reply:
```suggestion
/// Statistics are used in codec detection and stored in the fast field footer.
```